src/main/java/core/basesyntax/service/impl/FruitListImpl.java

```diff
@@ -0,0 +1,22 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.db.FruitTransaction;
+import core.basesyntax.service.FruitList;
+import java.util.ArrayList;
+import java.util.List;
+
+public class FruitListImpl implements FruitList {
+    @Override
+    public List<FruitTransaction> getFruitList(List<FruitTransaction> list) {
```

Comment:
The method 'getFruitList' is a bit misleading since it returns a list of 'FruitTransaction' objects rather than fruits. Consider renaming it to 'createFruitTransactionList' or something similar that better describes its functionality.